README.md

```diff
@@ -1,62 +1,66 @@
 
 LAME is a high quality MPEG Audio Layer III (MP3) encoder licensed under the LGPL.
 
 The packages generated with this **conanfile** can be found on [Bintray](https://bintray.com/bincrafters/public-conan/libmp3lame%3Abincrafters).
 
-## For Users: Use this package
+
+## Issues
+
+If you wish to report an issue or make a request for a package, please do so here:
+
+[Issues Tracker](https://github.com/bincrafters/community/issues)
+
+
+## For Users
 
 ### Basic setup
 
-    $ conan install libmp3lame/3.100@bincrafters/testing
+    $ conan install libmp3lame/3.100@bincrafters/stable
 
 ### Project setup
 
 If you handle multiple dependencies in your project is better to add a *conanfile.txt*
 
     [requires]
-    libmp3lame/3.100@bincrafters/testing
+    libmp3lame/3.100@bincrafters/stable
 
+    [generators]
+    txt
 
 Complete the installation of requirements for your project running:
 
     $ mkdir build && cd build && conan install ..
 
 Note: It is recommended that you run conan install from a build directory and not the root of the project directory.  This is because conan generates *conanbuildinfo* files specific to a single build configuration which by default comes from an autodetected default profile located in ~/.conan/profiles/default .  If you pass different build configuration options to conan install, it will generate different *conanbuildinfo* files.  Thus, they should not be added to the root of the project, nor committed to git.
 
-## For Packagers: Publish this Package
-
-The example below shows the commands used to publish to bincrafters conan repository. To publish to your own conan respository (for example, after forking this git repository), you will need to change the commands below accordingly.
 
 ## Build and package
 
 The following command both runs all the steps of the conan file, and publishes the package to the local system cache.  This includes downloading dependencies from "build_requires" and "requires" , and then running the build() method.
 
-    $ conan create bincrafters/testing
+    $ conan create . bincrafters/stable
 
 
 ### Available Options
 | Option        | Default | Possible Values  |
 | ------------- |:----------------- |:------------:|
-| fPIC      | True |  [True, False] |
 | shared      | False |  [True, False] |
+| fPIC      | True |  [True, False] |
+
 
 ## Add Remote
 
     $ conan remote add bincrafters "https://api.bintray.com/conan/bincrafters/public-conan"
 
-## Upload
-
-    $ conan upload libmp3lame/3.100@bincrafters/testing --all -r bincrafters
-
 
 ## Conan Recipe License
 
 NOTE: The conan recipe license applies only to the files of this recipe, which can be used to build and package libmp3lame.
 It does *not* in any way apply or is related to the actual software being packaged.
 
-[MIT](https://github.com/bincrafters/conan-libmp3lame.git/blob/testing/3.100/LICENSE)
+[MIT](https://github.com/feliwir/conan-libmp3lame/blob/stable/3.100/LICENSE.md)
```

Comment:
this is wrong

Reply:
Used the readme generator, but you are right of course :)

Reply:
The readme generator is written in a way that it can be used by others outside of bincrafters as well. I believe that there is a parameter in which you can specify the username which should get used